server/views/pages/recommendations/ppcs/indeterminateSentence/selectIndeterminatePpudSentence.njk

```diff
@@ -40,37 +40,39 @@
         <div class="govuk-grid-column-two-thirds-from-desktop">
             {% include '../../../../partials/error-summary.njk' %}
 
-            <p class='govuk-body'>Check the details of your booking then add it to the correct sentence in PPUD. If the
-                right sentence is not listed, select "add a sentence".</p>
+            <p class='govuk-body govuk-!-margin-bottom-8'>Check the details of your booking then add it to the correct sentence in PPUD.</p>
 
-            <h2 class='govuk-heading-m govuk-!-margin-top-4 govuk-!-margin-bottom-4'>Booking details from Part A</h2>
+            <h2 class='govuk-heading-m'>Booking details from Part A</h2>
 
+            {# Increase the bottom margin, as otherwise the separation between Part A and PPUD data isn't as obvious #}
             {{ summaryList({
                 id: 'nomis-sentence-details',
+                classes: 'govuk-!-margin-bottom-9',
                 rows: [
                     { key: 'Offence', value: hyphenForBlank(nomisSentence.offenceDescription) },
                     { key: 'Date of sentence', value: hyphenForBlank(formatDateTimeFromIsoString({ isoDate: nomisSentence.dateOfSentence })) },
                     { key: 'Sentence type', value: hyphenForBlank(nomisSentence.custodyGroup) },
                     { key: 'Sentence expiry date', value: hyphenForBlank(formatDateTimeFromIsoString({ isoDate: nomisSentence.sentenceExpiryDate })) }
-                ],
-                borderTop: true
+                ]
```

Comment:
Is there no longer a top border in the design for this page?

Reply:
No, it was removed: see https://dsdmoj.atlassian.net/browse/MRD-2685 and https://www.figma.com/design/9ECmQnLKoCZzj12YT5vDsD/PPCS-journey-v2?node-id=13210-11668&t=HnMh1YuwKQjV7cUA-4